R/process_extract_gp_ooh.R

```diff
@@ -82,6 +82,10 @@ process_extract_gp_ooh <- function(year, data_list, write_to_disk = TRUE) {
     dplyr::mutate(
       record_keydate1 = as.Date(.data$record_keydate1),
       record_keydate2 = as.Date(.data$record_keydate2)
+    ) %>%
+    # change attendance_status into numeric
+    dplyr::mutate(
+      attendance_status = as.double(attendance_status)
```

Comment:
I thought we had already resolved this... not sure how the episode files have been running properly if not?

That said, issue #625 relates to this but in the meantime, rather than introducing new code here, could we just make sure it's read in as a numeric for GP OoHs.

- [ ] Change [this line](https://github.com/Public-Health-Scotland/source-linkage-files/blob/b1442a01e3ef79783edb460e48416812cc4b15e5/R/read_extract_ooh_consultations.R#L16) to  `"Consultation Recorded" = readr::col_integer()`.
- [ ] Remove [this block of code](https://github.com/Public-Health-Scotland/source-linkage-files/blob/b1442a01e3ef79783edb460e48416812cc4b15e5/R/process_extract_consultations.R#L39).

Reply:
I was just going to make this change but realised it should already be a numeric by this point... So I think this bit of code can just be removed